packages/web/app/src/lib/preflight-sandbox/shared-types.ts

```diff
@@ -28,12 +28,10 @@ export namespace IFrameEvents {
       log: string | Error;
     };
 
-    type ResultEventData = {
+    export interface ResultEventData extends Omit<WorkerEvents.Outgoing.EventData.Result, 'type'> {
```

Comment:
I have opted to encode the passthrough relationship from the worker to the iframe with regards to result data.

Reply:
I will revert.

However I would like to clarify that DRY was not the point here.

This was about encoding a data relationship. This code would tell the reader that Iframe Result type is tied to the Worker Result type. It would imply that there is a passthrough relationship.

I am interpreting your feedback to mean the following: Any apparent data symmetry between these layers should be treated as incidental and not to be encoded.